reporters_db/data/reporters.json

```diff
@@ -19215,6 +19293,9 @@
             "name": "Southern Reporter",
             "variations": {
                 "(So. 2nd)": "So. 2d",
+                "S.": "So.",
+                "S. (2nd)": "So. 2d",
+                "SO.": "So.",
                 "So (2nd)": "So. 2d",
                 "So .2d": "So. 2d",
```

Comment:
This is a bad variation ... I point out because I shouldn't have added it ... I think.

Reply:
Some citations come with SO. instead of So.
Here is an example: https://cite.case.law/pdf/2917490/Davis%20v.%20McCain,%20171%20La.%201011,%20132%20SO.%20758%20(1931).pdf

Also So. comes as S.
https://cite.case.law/pdf/1225770/State%20ex%20rel.%20Tullidge%20v.%20Hollingsworth,%20108%20Fla.%20607,%20146%20S.%20660%20(1933).pdf

Reply:
I think that comes from a old change: https://github.com/freelawproject/reporters-db/commit/451c524adefd05f22bfa41bee06dc2e8f0d15447#diff-45f9b4fa9d04419349ec0fd08c75b92af5cdb7d89cd1c0c27194fa152f249e62R17285

It shouldn't go there right? I'll remove it

Reply:
Probably should've been fixed in opinionated. not here.  Thanks if you can remove it that would be great.